radicle-cli/examples/rad-issue.md

```diff
@@ -0,0 +1,32 @@
+Project 'todo' items are called 'issue's.  They can be inspected and modified
+using the 'issue' subcommand.
+
+Let's say the new car you are designing with your peers has a problem with its flux capacitor.
+
+```
+$ rad issue new --title "flux capacitor underpowered" --description "Flux capacitor power requirements exceed current supply"
+```
+
+The issue is now listed under our project.
+
+```
+$ rad issue list
+522dcca93e8e172fcbd0d661326527949f2967ed flux capacitor
+```
+
+Great! Now we've documented the issue for ourselves and others.
+
+Just like with other project management systems, the issue can be assigned to
+others to work on.  This is to ensure work is not duplicated.
+
+Let's assign ourselves to this one.
+
+```
+$ rad assign 522dcca93e8e172fcbd0d661326527949f2967ed z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi
+```
+
+Note: this can always be undone with the 'unassign' subcommand.
+
+```
+$ rad unassign 522dcca93e8e172fcbd0d661326527949f2967ed z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi
```

Comment:
@cloudhead how can I make these issue and peer ID's in the example reproducible?

Reply:
Check `radicle-cli/tests/commands.rs`

However, for COB IDs it might require something more, but at least ensure the user public key is stable, as that's one of the inputs to the COB ID.

Reply:
Looking at the code, it seems like the ObjectId of the cob is the root commit `oid`. This means commit timestamps probably affects the ID. This is a bit tricky, as we'll need a way to force the timestamp to a certain value to make it deterministic.